.editorconfig

```diff
@@ -1,18 +1,18 @@
-# EditorConfig is awesome: https://EditorConfig.org
-
-# top-most EditorConfig file
-root = true
-
-# Unix-style newlines with a newline ending every file
-[*]
-end_of_line = lf
-insert_final_newline = true
-
-# 4 space indentation
-[*.py]
-indent_style = space
-indent_size = 4
-
-# Tab indentation (no size specified)
-[Makefile]
-indent_style = tab
```

Comment:
@ayyubibrahimi This file shows all lines as being replaced, which is probably caused by EditorConfig extension not installed because I think otherwise VSCode on Windows automatically replace trailing LF with CR LF. So keep a look out for files like these.

Reply:
Will look into it tonight.

Reply:
It was installed, but I hadn't generated the file. Please see the file that was generated.

Reply:
@ayyubibrahimi please restore the `.editorconfig` file that I created originally. That was not what I meant.